google/cloud/spanner_dbapi/connection.py

```diff
@@ -183,6 +183,21 @@ def read_only(self, value):
             )
         self._read_only = value
 
+    @property
+    def request_options(self):
+        """Options for the next SQL operations.
+
+        Returns:
+            google.cloud.spanner_v1.RequestOptions:
+                Request options.
+        """
+        if self.request_priority is None:
+            return
+
+        req_opts = RequestOptions(priority=self.request_priority)
+        self.request_priority = None
+        return req_opts
```

Comment:
Request options are used only for next one request

Reply:
Is there a reason for this?

Reply:
Spanner is designed this way, otherwise there is no sense in request options.
Let's say there is a request priority. If it'll be set for a connection/session, than all the requests will have the same priority. It's technically the same that not setting priority at all.